src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -32,13 +32,41 @@ public ValueTuple(T1 item1, T2 item2)
 
         public override string ToString()
         {
-            return '{' + Item1?.ToString() + "", "" + Item2.ToString() + '}';
+            return '{' + Item1?.ToString() + "", "" + Item2?.ToString() + '}';
         }
     }
 }
 
             ";
 
+        private static readonly string trivial3uple =
+                @"
+
+    // PROTOTYPE: put in correct namespace
+    namespace System.Runtime.CompilerServices
+    {
```

Comment:
This should be the next change, since we seem to have settled on the namespace. This would need to be matched by changing WellKnown stuff